android/OnesignalPlugin.java

```diff
@@ -198,15 +198,15 @@ public static JSONObject get(OSNotification notification) {
       JSONObject additional_data;
 
       try {
-        os_data.put("notification_id", payload.notificationId);
+        os_data.put("notification_id", payload.notificationID);
         os_data.put("is_foreground", notification.isAppInFocus);
         os_data.put("title", payload.title);
         os_data.put("body", payload.body);
         additional_data = payload.additionalData;
         if (additional_data != null) {
           os_data.put("additional_data", additional_data.toString());
         }
-        os_data.put("launch_url", payload.launchUrl);
+        os_data.put("launch_url", payload.launchURL);
```

Comment:
game code doesn't need to change. the onesignal android sdk has updated their API in the latest versions to match with the iOS versions.

Older versions of the SDK were using launchUrl, newer versions would use launchURL